etl/steps/data/garden/wb/2025-07-01/income_groups.meta.yml

```
@@ -5,16 +5,17 @@ definitions:
         - Economic Growth
     processing_level: major
     description_from_producer: |-
-      For the current 2025 fiscal year, low-income economies are defined as those with a GNI per capita, calculated using the [World Bank Atlas method](https://datahelpdesk.worldbank.org/knowledgebase/articles/378832-what-is-the-world-bank-atlas-method), of $1,145 or less in 2023; lower middle-income economies are those with a GNI per capita between $1,146 and $4,515; upper middle-income economies are those with a GNI per capita between $4,516 and $14,005; high-income economies are those with more than a GNI per capita of $14,005.
+      For the current 2026 fiscal year, low-income economies are defined as those with a GNI per capita, calculated using the [World Bank Atlas method](https://datahelpdesk.worldbank.org/knowledgebase/articles/378832-what-is-the-world-bank-atlas-method), of $1,135 or less in 2024; lower middle-income economies are those with a GNI per capita between $1,136 and $4,495; upper middle-income economies are those with a GNI per capita between $4,496 and $13,935; high-income economies are those with more than a GNI per capita of $13,935.
 
       Please note: Regions in this table include economies at all income levels. The term country, used interchangeably with economy, does not imply political independence but refers to any territory for which authorities report separate social or economic statistics. Click [here](https://datahelpdesk.worldbank.org/knowledgebase/articles/378834-how-does-the-world-bank-classify-countries) for information about how the World Bank classifies countries.
+
     description_key:
       - The World Bank creates a yearly classification of countries by income, for all countries with population over 30,000.
       - This classification stays the same throughout the fiscal year (from July 1 to June 30) even if the income data for a country changes.
```

maybe it makes sense to clarify this some more:
```suggestion
      - This classification stays the same throughout the World Bank's fiscal year (from July 1 to June 30) even if the income data for a country changes.
```